learning.md

```diff
@@ -67,6 +67,15 @@ Table of Contents
  * [Udemy Apache Kafka Series - Confluent Schema Registry & REST Proxy](https://goo.gl/XgWcVz) - Intermediate
 
 ## Blogs
+ * [Use External Schema Registry With MSK Connect – Part 1 Local Development](https://jaehyeon.me/blog/2022-03-07-schema-registry-part1/)
```

Comment:
Please only mention your blog once. If you have a tag or category that specifically mentions kafka that's the best way to link to it.